src/main/java/com/amazonaws/services/dynamodbv2/transactions/Request.java

```diff
@@ -444,11 +451,33 @@ protected static Request deserialize(String txId, ByteBuffer rawRequest) {
 
         @JsonIgnore
         public abstract RequestClientOptions getRequestClientOptions();
-        
+
+        @JsonIgnore
+        public abstract void setGeneralProgressListener(ProgressListener progressListener);
+
+        @JsonIgnore
+        public abstract void setProgressListener(ProgressListener progressListener);
+
     }
-    
+  
+    private static abstract class ExpectedAttributeValueMixIn {
+      
+      @JsonIgnore
+      public abstract void setComparisonOperator(String comparisonOperator);
```

Comment:
might also need to JsonIgnore
setComparisonOperator(ComparisonOperator comparisonOperator)